composer.json

```diff
@@ -11,18 +14,18 @@
         {
             "name": "Phil Bennett",
             "email": "philipobenito@gmail.com",
-            "homepage": "http://philipobenito.github.io",
+            "homepage": "http://www.philipobenito.com",
             "role": "Developer"
         }
     ],
     "require": {
-        "php" : ">=5.4.0",
-        "league/container": "~1.0",
-        "nikic/fast-route": "~0.3",
-        "symfony/http-foundation": "~2.6"
+        "league/container": "^2.0.2",
```

Comment:
@shadowhand it includes the code so that the package can be more easily used out of the box. You are aware that this doesn't require you to use the code included if you do not want to?